pyproject.toml

```diff
@@ -29,6 +29,7 @@ joblib = "^1.2.0"
 matplotlib = "^3.7.0"
 psutil = "^5.9.4"
 tabulate = "^0.9.0"
+hyperopt = "^0.2.6"
```

Comment:
I don't know if it is worth to keep `hyperopt` here. Perhaps it is better to have it as extra dep